opi

```diff
@@ -54,13 +57,13 @@ if ($serialized_query =~ m/skype/ ) {
 #
 # API configuration
 #
-my $obs_apiroot = 'https://api.opensuse.org';
+my $_obs_apiroot = 'https://api.opensuse.org';
```

Comment:
The underscore prefix should be added to the variable in function, not global variable.

Reply:
I chose the global because it made the variable line out better. Do you want a new PR with these two variable names swapped?

Reply:
The best way is to avoid underscore. Global variables can be renamed to `$opensuse_obs_apiroot` and `$packman_obs_apiroot`.